canjs-devtools-injected-script.js

```diff
@@ -519,11 +519,17 @@
 
 		getViewModelKeys(viewModel) {
 			if (canReflect.isListLike(viewModel)) {
-				return canReflect.getOwnEnumerableKeys(viewModel);
+				return canReflect.getOwnEnumerableKeys(viewModel).filter(key => {
+					if (typeof key === 'string') {
```

Comment:
Can `getOwnEnumerableKeys` return non-strings? If it can, can you make this check `return typeof key === ‘string’ && key.startsWith(‘_’) === false;`?